include/mbedtls/rsa.h

```diff
@@ -181,6 +181,27 @@ void mbedtls_rsa_init(mbedtls_rsa_context *ctx);
 int mbedtls_rsa_set_padding(mbedtls_rsa_context *ctx, int padding,
                             mbedtls_md_type_t hash_id);
 
+/**
+ * \brief          This function retrieves padding mode of RSA modulus.
+ *
+ * \param ctx      The initialized RSA context.
+ *
+ * \return         RSA padding mode.
+ *
+ */
+int mbedtls_rsa_get_padding_mode(const mbedtls_rsa_context *ctx);
```

Comment:
I prefer to using `mbedtls_rsa_get_padding_mode` because `mbedtls_rsa_set_padding` sets `padding mode` plus `hash_id`. That's why I didn't use `mbedtls_rsa_get_padding` initially.